include/envoy/server/admin.h

```diff
@@ -86,9 +86,9 @@ class Admin {
    * its data.
    * @return Http::Code the response code.
    */
-  using HandlerCb = std::function<Http::Code(
-      absl::string_view path_and_query, Http::ResponseHeaderMap& response_headers,
-      Buffer::Instance& response, AdminStream& admin_stream)>;
+  using HandlerCb = std::function<Http::Code(absl::string_view path_and_query,
+                                             Http::ResponseHeaderMap& response_headers,
+                                             Buffer::Chunker& response, AdminStream& admin_stream)>;
```

Comment:
s/Buffer::Chunker/Chunker/ here and elsewhere.

Reply:
Get it.